### PR TITLE
docs: vmanomaly slight improvements

### PR DESCRIPTION
**Pull request by fred-navruzov**
- better messaging
- update links to dockerhub in guides
- added anomaly_score to FAQ
- improve model section (sort + use cases)
- slight refactor of a guide (vmanomaly+vmalert)
- add /docs/.jekyll-metadata to .gitignore

### CODE REVIEW

**Comment by tenmozes** [comment]
you don't need .jekill metadata file in this PR

File: docs/anomaly-detection/guides/guide-vmanomaly-vmalert.md [hunk 1]
```diff
@@ -1,17 +1,18 @@
 ---
 weight: 1
 sort: 1
-title: Getting started with vmanomaly
+title: vmanomaly integration
```
Comment (tenmozes) [inline docs/anomaly-detection/guides/guide-vmanomaly-vmalert.md:4]: please rename it as well here - https://github.com/VictoriaMetrics/VictoriaMetrics/tree/master/deployment/docker/vmanomaly/vmanomaly-vmalert-guide with folder (in that case you need to change links too) or/and Readme.md

Comment (fred-navruzov) [reply docs/anomaly-detection/guides/guide-vmanomaly-vmalert.md:4]: thanks, renamed folder, changed ref, updated title reference

**Comment by hagen1778** [comment]
> you don't need .jekill metadata file in this PR

should we add it to .gitignore?

**Pull request merged**